python/ray/tests/test_scheduling.py

```diff
@@ -519,6 +519,64 @@ def f(s, ref):
     wait_for_condition(lambda: not fetches_queued())
 
 
+def build_cluster(num_cpu_nodes, num_gpu_nodes):
+    cluster = ray.cluster_utils.Cluster()
+    gpu_ids = [
+        cluster.add_node(num_cpus=10, num_gpus=1).unique_id
```

Comment:
This is a lot of workers (5 * 10 = 50), which might cause instability in CI. How about reducing this n / num cpus?

Reply:
done